src/Agent/NewRelic/Agent/Core/Config/Configuration.cs

```diff
@@ -1,6 +1,3 @@
-// Copyright 2020 New Relic, Inc. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Alex and I have been manually adding this copyright back in... not sure if it matters though

Reply:
Fixed!